bc/core/utils/network.py

```diff
@@ -18,6 +20,13 @@ def strip_port_to_make_ip_key(ip_str: str | None) -> str | None:
     :param ip_str: the IP address of the viewer and the source port of the request
```

Comment:
The docstring needs a few tweaks here.

Reply:
Thanks!. I'm going to update the docstring.